dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -13,41 +15,46 @@ LICENSE="MIT public-domain"
 SLOT="0/1.10.0"
 KEYWORDS="~amd64 ~arm ~x86"
 
-IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
+IUSE="beta-components +cache cg doc double-precision egl examples +freeimage
+	gles2 ois +opengl profile +resman-legacy resman-pedantic resman-strict
+	tools"
 
-REQUIRED_USE="examples? ( ois )
-	gles3? ( gles2 )
-	gl3plus? ( opengl )"
+REQUIRED_USE="
+	|| ( gles2 opengl )
+	^^ ( resman-legacy resman-pedantic resman-strict )
+	examples? (
+		beta-components
+		ois
```

Comment:
likewise, what do we gain from a separate `ois` flag except added complexity? it does not tick a build switch, and only serves a role in this `REQUIRED_USE` condition, when `examples` could be used to pull in the dependency just as well.

Reply:
And what if a user does not need the examples but wants to use input streams without putting dev-games/ois into their world file?

Reply:
> And what if a user does not need the examples but wants to use input streams without putting dev-games/ois into their world file?

As an optional runtime dependency it should be announced in pkg_postinst, having it in the world file is perfectly acceptable. But I don't mind it as a permanent `RDEPEND` either, if it does not add a significant dependency chain.

Reply:
Also the description for ois may be slightly misleading if it really is used for something else than samples: `pull in Object-oriented Input System library for samples`

Reply:
Hmmm... a question would be who actually would _not_ want to install ois...

It only depends on x11-libs/libXaw and x11-libs/libX11. Both will be installed anyway. And ois itself brings only about 1.2MB to the table, so I think making it a permanent dependency is in order.